cmake/targets/BuildTinyCC.cmake

```diff
@@ -1,15 +1,15 @@
 register_repository(
   NAME
-    tinycc
```

Comment:
Are you using a formatter? The spacing is intentional for readability

Reply:
yeah, fixed